yellowbrick/datasets/download.py

```diff
@@ -17,13 +17,8 @@
 ##########################################################################
 
 import os
-import six
 import zipfile
 
-if six.PY2:
-    # backport for encoding in open for python2
-    from io import open
-
 try:
```

Comment:
Great catch on the six stuff! In addition to dropping that, I think we can also safely change the try-except `urllib` import in this same file to just `from urllib.request import urlopen`